submissions/asaMitaka/frogger/app.js

```diff
@@ -0,0 +1,169 @@
+// Enemies our player must avoid
+var newP = document.createElement('p');
```

Comment:
You should avoid using `var`s. Most of the time to store a value `const` is enough and only if you are 100% sure that you'll change this value in code later use `let`.